### PR TITLE
add matt (maintainer of bentley SaaS)

### PR DESCRIPTION
**Pull request by mattyokan**
thank you in advance

(CEO of Bentley SaaS - newest unicorn startup)

### CODE REVIEW

**Comment by mattyokan** [comment]
Requesting a code review from @Penple so we can better understand the implications of this large change. Really feeling much better about this revised timeline after the Monday morning leadership sync-up. Glad we could identify some clear action items to get us back to green.

**Comment by Penple** [comment]
> Requesting a code review from @Penple so we can better understand the implications of this large change. Really feeling much better about this revised timeline after the Monday morning leadership sync-up. Glad we could identify some clear action items to get us back to green.

Who are you?

**Comment by Aberdeener** [comment]
Hi

**Comment by deanveloper** [comment]
not with betnly pictrs

**Comment by Penple** [comment]
> not with betnly pictrs

Hello?

**Comment by Aberdeener** [comment]
Why

File: pages/index.tsx [hunk 1]
```diff
@@ -78,6 +78,10 @@ const people = [{
 	name: "violet",
 	color: "#ffaaff",
 	website: "https://viomck.com/"
+}, {
+        name: "matt",
+        color: "#44eafc",
+        website: "https://bentley.pictures/"
```
Comment (Aberdeener) [inline pages/index.tsx:84]:
```suggestion
        website: "https://oplegends.com/?ref=analyse.net"
```

Comment (Penple) [reply pages/index.tsx:84]: perfect

**Pull request closed without merging**